@xen-orchestra/web-core/lib/components/table/UiTable.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <table :class="{ 'vertical-border': verticalBorder, error: color === 'error' }" class="ui-table typo p2-regular">
```

Comment:
Based on my comment for the `hasError` prop:
```suggestion
  <table :class="{ 'vertical-border': verticalBorder, error: hasError }" class="ui-table typo p2-regular">
```